### PR TITLE
Add stub of help functionality

### PR DESCRIPTION
**Pull request by sfhardman**

### CODE REVIEW

File: client/src/components/shared/Header.js [hunk 1]
```diff
@@ -1,176 +1,83 @@
...
+        true, 2));
+      helpAnchor = string(`edit ${crumbTitle}`).slugify().toString();
+    }
+    if ((!helpAnchor) && (splitPath.length > 1) && (splitPath[1] === 'edit')) {
+      helpAnchor = 'edit';
     }
   }
 
-  const menuElements = makeMenuItems(menuItems, location);
+  console.log(currentRoute);
```
Comment (blackwidow207) [inline client/src/components/shared/Header.js]: @sfhardman 
Did you mean to leave these 2 console logs here?

Comment (sfhardman) [reply client/src/components/shared/Header.js]: Nope, will remove them now...

Comment (sfhardman) [reply client/src/components/shared/Header.js]: Done

**Review by blackwidow207 — APPROVED** [review]
reviewed changes, looks good to me

**Pull request merged**